webclient/src/dialogs/KnownHostDialog/KnownHostDialog.i18n.json

```diff
@@ -0,0 +1,6 @@
+{
+ "KnownHostDialog": {
+    "title": "{mode, select, edit {Edit} other {Add}} Known Host",
```

Comment:
@ebbit1q here is an example using the `select` operator, similar to the `plural` operator.